Model/Config.php

```diff
@@ -316,5 +318,35 @@ public function isExpressPayEnabled(int $websiteId): bool
             ScopeInterface::SCOPE_WEBSITES,
             $websiteId
         );
+    }    
+    
+    /**
+     * Check if Express Pay buttons are enabled.
```

Comment:
[nit]: might be good to update the descriptions in this and the next doc block to be specific to these new settings

Reply:
I agree, we should probably get the design team to come up with updated mocks and add it as a capstone to the feature release

Reply:
That's probably a good idea too, for this comment I was just meaning the comment blocks in the config file. Really just a tiny nitpick as it doesn't effect any functionality.

Reply:
ahhh my mistake, updated!